base/files/file_enumerator.h

```diff
@@ -20,7 +20,7 @@
 #if defined(STARBOARD)
 #include <sys/stat.h>
 #include <unistd.h>
-#include "starboard/file.h"
+
```

Comment:
Small nit, but you have a blank line left in a few places here where we don't need it. I'll just add "remove" below for those.

Reply:
:( I thought the linter would take care of those, will fix!